katalog/etcd-backup-s3/kustomization.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kube-system
+
+resources:
+  - cronjob.yaml
+
+images:
+  - name: etcd
+    newName: registry.sighup.io/fury/etcd-backup/etcd
+    newTag: 3.5.17
```

Comment:
We are using 3.5.15 in the on-prem installer, should this be aligned?
https://github.com/sighupio/fury-kubernetes-on-premises/blob/35b4b46177bfe7ed17d424007a4416c87effd093/roles/etcd/defaults/main.yml#L3

Also, etcd version could be different in the 3 maintained releases of KFD, for example 1.28 maybe uses 3.5.x and 1.29 3.6.y

Reply:
I think that etcdctl just needs to use the correct version of the API, we only use the CLI, right @kriive ?

Reply:
That's a great point. I believe that the snapshot interface is stable (it is a [gRPC call](https://github.com/etcd-io/etcd/blob/3c916bb2587174008054f629d784514abe9e8d36/client/v3/maintenance.go#L230)), so the actual patch or minor version we're on doesn't really matter, as long as we keep compatibility with the major version.